README.md

```diff
@@ -0,0 +1,44 @@
+# Dapr Conformance tests
+
+A github action for running conformance test against a pluggable component.
+
+```yaml
+- name: Conformance Tests
+  uses: dapr-sandbox/action-components-conformance-test@v0.1.0
+  with:
+    type: state
+    socket: /tmp/socket.sock
+    metadata: | ## component-specific init metadata
+      timeout: 10s
+    operations: | ## If not provided all operations will be tested
```

Comment:
Nit: provide examples of more than 1 operation so it's easier to see how it's used